### ISSUE TITLE
[docs] Search result points to the wrong link if selected with enter instead of mouse

### ISSUE THREAD

**Issue by vuki656** [dotansimha/graphql-code-generator#5623]
**Describe the bug**
When searching for `introspection` or any other query in the docs, if you press enter to select the result it points to an incorrect link `https://graphql-code-generator.com/https://graphql-inspector.com/docs/essentials/introspect`
 

**To Reproduce**

1. Go to the docs [https://graphql-code-generator.com/](https://graphql-code-generator.com/)
2. In the search type in `introspection`
3. Press enter to select the first result 
4. It will take you to `https://graphql-code-generator.com/https://graphql-inspector.com/docs/essentials/introspect`

**Expected behavior**
It should go to the correct link even if selected with entering key instead of pressed with a mouse

**Environment:**

- OS: Ubuntu 20.4
- Chome: Version 88.0.4324.182 (Official Build) (64-bit)

**Additional context**
Didn't test it on any other browsers but I assume its the same

**Comment by ardatan** [comment]
Thanks for reporting this!
@tvvignesh Do you have any idea why this happens?

**Comment by tvvignesh** [comment]
@vuki656 Thanks for reporting this. It is a bug which got introduced due to one of my commits. Will send a PR to fix this soon.

@ardatan Its due to this change: https://github.com/dotansimha/graphql-code-generator/blob/master/patches/%40docsearch%2Breact%2B3.0.0-alpha.32.patch#L9 where I got rid of the transform to support multi-site search.

Docsearch prefixes every link with the base url which I got rid off but looks like they have different behavior when pressing enter key 🤔
